src/main/java/com/sap/sgs/phosphor/fosstars/model/value/Languages.java

```diff
@@ -120,7 +120,8 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return elements.stream().map(Enum::toString).collect(Collectors.joining(", "));
+    return elements.stream()
```

Comment:
There seems to be no difference in the line.

Reply:
I added sorted() call. But looks like this call is not necessary anymore since the class uses TreeSet. I'll remove it.